core/geometry/src/topology/HalfEdgePointInGraphSearch.ts

```diff
@@ -95,88 +96,109 @@ export class PointSearchContext {
+   * the new position and updates `searchBase` accordingly.
+   * @param searchBase the detail to reposition.
+   * @param ray the ray to use for repositioning. Ray is assumed to start at the vertex precisely.
+   * @param targetDistance the target distance to aim for.
+   */
   public reAimFromVertex(
-    searchBase: HalfEdgePositionDetail,
-    ray: Ray3d,
-    targetDistance: number): HalfEdgePositionDetail {
+    searchBase: HalfEdgePositionDetail, ray: Ray3d, targetDistance: number,
+  ): HalfEdgePositionDetail {
     const vertexNode = searchBase.node;
     let result;
     let outboundEdge = vertexNode!;
     do {
       // DPoint3d xyzBase;
-      // vu_getDPoint3d(& xyzBase, outboundEdge);
+      // vu_getDPoint3d(&xyzBase, outboundEdge);
       const data0 = NodeXYZUV.createNodeAndRayOrigin(outboundEdge.faceSuccessor, ray);
       const data1 = NodeXYZUV.createNodeAndRayOrigin(outboundEdge.facePredecessor, ray);
       const u0 = data0.u;
-      // double u1 = data1.GetU ();
+      // double u1 = data1.u;
       const v0 = data0.v;
       const v1 = data1.v;
-      if (Math.abs(v0) < this._tol) {
+      if (Math.abs(v0) < this._tol) { // if ray is parallel to outboundEdge.faceSuccessor
         if (Math.abs(u0 - targetDistance) < this._tol) {
-          // Direct hit at far end
+          // direct hit at far end
           result = searchBase.resetAsVertex(data0.node);
           result.setITag(1);
           return result;
         } else if (u0 > targetDistance) {
-          // Direct hig within edge
+          // direct hit within edge
           const edgeFraction = targetDistance / u0;
           result = searchBase.resetAtEdgeAndFraction(outboundEdge, edgeFraction);
           return result;
         } else if (Math.abs(u0) <= this._tol) {
-          // Unexpected direct hit on the base of the search, but call it a hit....
+          // unexpected direct hit on the base of the search, but call it a hit
           result = searchBase.resetAsVertex(outboundEdge);
           result.setITag(1);
           return result;
         } else if (u0 > this._tol) {
-          // Advance to vertex  ...
+          // advance to vertex
           // double edgeFraction = targetDistance / u0;
           result = searchBase.resetAsVertex(data0.node);
           return result;
         } else {
-          // Search direction is exactly opposite this edge.
-          // See if the other side of the sector is turned even beyond that ...
+          // search direction is exactly opposite this edge
+          // see if the other side of the sector is turned even beyond that
           if (v1 > this._tol) {
             result = searchBase.resetAsFace(outboundEdge, outboundEdge);
             return result;
           }
         }
-      } else if (v0 < -this._tol) {
+      } else if (v0 < -this._tol) { // if sweep from ray to outboundEdge.faceSuccessor is CCW
```

Comment:
Comment should be CW because v0 is negative
```suggestion
      } else if (v0 < -this._tol) { // if sweep from ray to outboundEdge.faceSuccessor is CW
```

Reply:
Addressed